packages/scene-graph-mediator/runtime/src/importer/Pixi.ts

```diff
@@ -294,13 +295,13 @@ export default class Pixi extends Importer {
       // TODO: support spine
       // object = new PIXI.spine.Spine(resources[node.id].data);
     } else if (node.sprite) {
-      // TODO: base64 image
-
       let texture = null;
       if (node.sprite.atlasUrl && node.sprite.frameName) {
         texture = PIXI.Texture.fromFrame(node.sprite.frameName);
       } else if (node.sprite.url) {
         texture = resources[node.sprite.url].texture;
+      } else if (node.sprite.base64) {
+        texture = PIXI.Texture.fromImage(node.sprite.base64);
```

Comment:
Supports base64 texture to make it easy to use texture for testing.

Reply:
thx!